Source/Server/Managers/ChatManager.cs

```diff
@@ -267,10 +285,14 @@ public static string GetUsernameFromMention(string mention)
             return mention.Replace("@", "");
         }
 
-        public static void ShowChatInConsole(string username, string message)
+        public static void ShowChatInConsole(string username, string message, bool fromDiscord = false)
         {
             if (!Master.serverConfig.DisplayChatInConsole) return;
-            else Logger.Message($"[Chat] > {username} > {message}");
+            else 
+            {
+                if (fromDiscord) Logger.Message($"[Discord] > {username} > {message}");
```

Comment:
Could add just a small enriched log message with a bit more context. (Isn't needed if bugs are not expected practically)

Logger.Message($"[Discord] {nameof(SendMessageToChatChannel)} - {username} > {message}");